### PR TITLE
use public VulnerableCode instance in VulnTotal 

### PR DESCRIPTION
**Pull request by keshav-space**
requires VCIO_TOKEN in .env file

Signed-off-by: Keshav Priyadarshi <git@keshav.space>

### CODE REVIEW

File: vulntotal/datasources/vulnerablecode.py [hunk 1]
```diff
@@ -85,3 +85,26 @@ def parse_advisory(fetched_advisory) -> VendorData:
...
         aliases=aliases, affected_versions=affected_versions, fixed_versions=fixed_versions
     )
+
+
+class VCIOTokenError(Exception):
+    pass
+
+
+def fetch_vulnerablecode_query(url: str, payload: dict):
+    load_dotenv()
```
Comment (TG1999) [inline vulntotal/datasources/vulnerablecode.py:95]: Any particular reason to use `load_dotenv()` here, we have accessed the environment variables without this as well in past for example here https://github.com/nexB/vulnerablecode/blob/main/vulnerabilities/utils.py#L238 ?

File: vulntotal/datasources/vulnerablecode.py [hunk 2]
```diff
@@ -85,3 +85,26 @@ def parse_advisory(fetched_advisory) -> VendorData:
...
+    vcio_token = os.environ.get("VCIO_TOKEN", None)
+    if not vcio_token:
+        msg = "Cannot call VulnerableCode API without a token set in the VCIO_TOKEN environment variable."
+        raise VCIOTokenError(msg)
+
+    response = (
+        requests.post(url, headers={"Authorization": vcio_token}, json=payload)
```
Comment (TG1999) [inline vulntotal/datasources/vulnerablecode.py:102]: Add a comment showing what is expected in token in .env , AFAIK either the user can store just the token and you have to do something like this in that case `headers={"Authorization": "Token f{vcio_token}"}` or add a comment explaining the env should have something like this : `Token <YOUR_VCIO_TOKEN>`

**Review by TG1999 — CHANGES_REQUESTED** [review]
Thakns!
minor nitpickings for your review

File: vulntotal/datasources/vulnerablecode.py (hunk 1 above)
Comment (keshav-space) [reply vulntotal/datasources/vulnerablecode.py:95]: VulnTotal is not a Django app, it cannot access the env variable loaded by Django. Hence the need to use `load_dotenv()`

**Review by TG1999 — APPROVED** [review]
LGTM! Thanks

**Pull request merged**